src/main/java/stirling/software/SPDF/config/security/oauth2/CustomOAuth2AuthenticationSuccessHandler.java

```diff
@@ -77,23 +74,22 @@ public void onAuthenticationSuccess(
                 throw new LockedException(
                         "Your account has been locked due to too many failed login attempts.");
             }
+
             if (userService.isUserDisabled(username)) {
                 getRedirectStrategy()
                         .sendRedirect(request, response, "/logout?userIsDisabled=true");
-                return;
             }
             if (userService.usernameExistsIgnoreCase(username)
                     && userService.hasPassword(username)
                     && !userService.isAuthenticationTypeByUsername(username, AuthenticationType.SSO)
                     && oAuth.getAutoCreateUser()) {
-                response.sendRedirect(contextPath + "/logout?oauth2AuthenticationErrorWeb=true");
-                return;
+                response.sendRedirect(contextPath + "/logout?oAuth2AuthenticationErrorWeb=true");
             }
+
             try {
                 if (oAuth.getBlockRegistration()
                         && !userService.usernameExistsIgnoreCase(username)) {
-                    response.sendRedirect(contextPath + "/logout?oauth2_admin_blocked_user=true");
-                    return;
+                    response.sendRedirect(contextPath + "/logout?oAuth2AdminBlockedUser=true");
                 }
```

Comment:
`return;` must be set, otherwise `java.lang.IllegalStateException: Committed` is thrown